processors/filterprocessor/expression/expression.go

```diff
@@ -0,0 +1,65 @@
+package expression
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// Type is the type of the filter (i.e. const, regex, etc)
+type Type string
```

Comment:
I think naming things "type" or any other language keyword is a bit confusing. Maybe `ExpressionFilterType` or `FilterType` or something like that?

Reply:
Done